psp-disable/README.md

```diff
@@ -1,12 +1,75 @@
-# Disable Pod Security Policies
 
-PSPs are deprecated as of Kubernetes 1.21 and are being removed as of Kubernetes v1.25.
+# Disabling Pod Security Policies 
 
-Use the [`disable-psps.yaml`](disable-psps.yaml) with your Helm deployment to disable them.
+Pod Security Policies (PSPs) are deprecated as of Kubernetes 1.21 and are being removed in Kubernetes v1.25. The deprecation may cause issues such as pods not spinning up, although  services, deployments, and replicasets are up and running. This can be a key indication of issues with PSPs, especially when running on an older version of Kubernetes.
 
+## Identifying PSP Issues
+If you suspect PSP issues due to pods not launching while other components appear unaffected, you can confirm this by describing your replicasets:
 ```shell
-helm upgrade --install kubecost --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer \
-  --namespace kubecost \
-  -f <your-other-values-files> \
-  -f ./disable-psps.yaml
+kubectl describe rs <replica-set-name>
 ```
+This command helps identify if PSPs are blocking pod creation due to insufficient or overly restrictive permissions.
+
+## Troubleshooting PSPs
+
+### Method 1: Simple Environment
+For clusters that do not have extensive security configurations:
+   
+ **Disable PSPs - Helm**
+   
+   Deploy or update Kubecost with PSP disabled using the [`disable-psps.yaml`](disable-psps.yaml) configuration:
+   ```shell
+   helm upgrade --install kubecost kubecost/cost-analyzer \
+       --namespace kubecost \
+       -f <your-other-values-files> \
+       -f ./disable-psps.yaml
+   ```
```

Comment:
The Kubecost version 2.x chart does not manage and does not support PSPs. The values referenced here no longer exist so this effectively does nothing.

Reply:
Then we should probably only keep my workaround that will help the users with older cluster versions to help me install Kubecost. Let me know if you need any more info !

Reply:
I think at this point, since we are on the eve of the fourth minor version of 2.x, we probably don't need this at all. It is very uncommon for users to be installing 1.x versions right now.